flow/scripts/global_route.tcl

```diff
@@ -31,6 +31,7 @@ proc global_route_helper {} {
     if {[expr !$::env(GENERATE_ARTIFACTS_ON_FAILURE) || \
         ![file exists $::global_route_congestion_report] || \
         [file size $::global_route_congestion_report] == 0]} {
+      write_sdc -no_timestamp $::env(RESULTS_DIR)/5_1_grt.sdc
```

Comment:
Why is this in the !$::env(GENERATE_ARTIFACTS_ON_FAILURE) branch?  You said you want it for GENERATE_ARTIFACTS_ON_FAILURE=1

Reply:
Failing eyesight, will fix

Reply:
Fixed.

I wish there was an easy and robust way to induce a global route failure in one of the ORFS designs...